src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DateTimePickerTests.cs

```diff
@@ -275,27 +275,50 @@ public void DateTimePicker_CalendarMonthBackground_SetInvalid_ThrowsArgumentExce
     public void DateTimePicker_BackColorChangedEvent_Raised_Success()
     {
         using DateTimePicker control = new();
-        bool eventWasRaised = false;
+        int callCount = 0;
+        EventHandler handler = (sender, e) =>
+        {
+            sender.Should().Be(control);
+            e.Should().Be(EventArgs.Empty);
+            callCount++;
+        };
 
-        control.BackColorChanged += (sender, args) => eventWasRaised = true;
+        control.BackColorChanged += handler;
         control.BackColor = Color.Red;
+        callCount.Should().Be(1);
 
-        eventWasRaised.Should().BeTrue();
+        control.BackColorChanged -= handler;
+        control.BackColor = Color.Green;
+        callCount.Should().Be(1);
     }
 
     [WinFormsFact]
     public void DateTimePicker_BackgroundImageChangedEvent_Raised_Success()
     {
         using DateTimePicker control = new();
-        bool eventWasRaised = false;
+        int callCount = 0;
+        EventHandler handler = (sender, e) =>
+        {
+            sender.Should().Be(control);
+            e.Should().Be(EventArgs.Empty);
+            callCount++;
+        };
 
-        control.BackgroundImageChanged += (sender, args) => eventWasRaised = true;
+        control.BackgroundImageChanged += handler;
         using (Bitmap bmp = new Bitmap(10, 10))
```

Comment:
Please use target type new
```suggestion
        using (Bitmap bmp = new(10, 10))
```